src/ListView.js

```diff
@@ -0,0 +1,311 @@
+import './ListView.css';
+import LineList from './LineList.js';
+import {useState} from 'react';
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+import { collection, query, setDoc, doc, updateDoc, deleteDoc, orderBy} from "firebase/firestore";
+import {useCollectionData} from "react-firebase-hooks/firestore";
+
+
+function ListView(props) {
+
+    // state data to be used later
+    const [selected, setSelected] = useState([]);
+    const [edited, setEdited] = useState(-1);
+    const [hidden, setHidden] = useState(false);
+    const [showWarning, setShowWarning] = useState(false);
+    const [showPriorities, setShowPriorities] = useState(false);
+    const [sort, setSort] = useState("creationAsc");
+    const [sortOptions, setSortOptions] = useState(false);
+    let showDeleteButton = selected.length > 0;
+    let disableChecks = (edited !== -1);
+
+    const qCreationAsc = query(collection(props.db, props.collection, props.listId, "Notes"));
+    const qCreationDesc = query(collection(props.db, props.collection, props.listId, "Notes"), orderBy("created", "desc"));
```

Comment:
Better to just store the orderBy component (since everything else is shared), and then have collectionSelector return query(collection(...), orderByTextAsc), for example.